docusaurus/docs-internals/editor-controls/data-controls/paypal.md

```diff
@@ -0,0 +1,202 @@
+| `label?`           | `string`                                                                                                                                                                        |    -    | The label displayed on the left side of the control                                                                                                                                                                               |
+| `position?`        | `number`                                                                                                                                                                        |    -    | The position of the control in toolbar                                                                                                                                                                                            |
+| `devices?`         | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                        | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`        | `boolean`                                                                                                                                                                       | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+| `helper?.content`  | `string`                                                                                                                                                                        |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                 |
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                |
+
+### Return value
+
+The return value is a `string` representing the text field value.
```

Comment:
review description, because we store the api-key

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/1fe3f7e4-e283-4757-aa4a-3dd12b61dadd)